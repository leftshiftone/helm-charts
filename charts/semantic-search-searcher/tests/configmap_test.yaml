suite: test configmap
templates:
  - configmap.yaml
tests:
  - it: default
    set:
      keycloak.enabled: true
    asserts:
      - isKind:
          of: ConfigMap
      - equal:
          path: metadata.name
          value: RELEASE-NAME-semantic-search-searcher
      - matchSnapshot:
          path: metadata.labels
      - matchSnapshot:
          path: data
