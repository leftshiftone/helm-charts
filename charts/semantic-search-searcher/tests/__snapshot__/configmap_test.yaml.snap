default:
  1: |
    app.kubernetes.io/instance: RELEASE-NAME
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: semantic-search-searcher
    app.kubernetes.io/version: 0.0.0
    helm.sh/chart: semantic-search-searcher-0.0.0
  2: |
    ENABLE_METRICS: "true"
    ES_URL: http://RELEASE-NAME-elasticsearch-es-http:9200
    HTTP_LOG_LEVEL: WARNING
    LOG_LEVEL: INFO
